src/main/java/org/folio/consortia/service/ConsortiaConfigurationService.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.consortia.service;
+
+public interface ConsortiaConfigurationService {
+  /**
+   * Get Config based on requested tenant.
+   * If process failed, it will specify error with tenant id.
+   * There will be only one record, so don't need any parameter
+   *
+   * @return central tenant id
+   */
+  String getCentralTenant();
```

Comment:
Lets use the same naming convention and rename to getCentralTenantId()

Reply:
fixed